packages/@n8n/api-types/src/schemas/project.schema.ts

```diff
@@ -20,7 +20,7 @@ export const projectRoleSchema = z.enum([
 export type ProjectRole = z.infer<typeof projectRoleSchema>;
 
 export const projectRelationSchema = z.object({
-	userId: z.string(),
-	role: projectRoleSchema,
+	userId: z.string().min(1),
+	role: projectRoleSchema.exclude(['project:personalOwner']),
```

Comment:
I've excluded the `project:personalOwner` here too. Personal projects are special, they can only have one owner with that role. There should not be any API endpoint that allows changing the relation. They are created when the user is created and deleted when the user is deleted.

Reply:
I've temporarily reverted this change, to make the build pass again.
Since there is a a lot more projects related types that are still duplicated between packages, I'd like to clean this up later in a separate PR as well.